calico_node/bgpconfig.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: projectcalico.org/v2
```

Comment:
could we move this perhaps to a `test` or `st` subdirectory?

Reply:
That should be removed. I think it got checked in accidentally.  I have some cleanup that I was working on that should remove it.

Reply:
It is a file generated from the tests running.

Reply:
OK, will remove.